api/tests/mocha/services/setup/utils.spec.js

```diff
@@ -928,48 +940,49 @@ describe('Setup utils', () => {
 
     it('should throw invalid url error', async () => {
       const payload = { containers: [], docker_compose: {} };
-      sinon.stub(rpn, 'post').resolves({});
+      sinon.stub(request, 'post').resolves({});
       process.env.UPGRADE_SERVICE_URL = 'whatever';
       utils = rewire('../../../../src/services/setup/utils');
 
-      await expect(utils.makeUpgradeRequest(payload)).to.be.rejectedWith('Invalid UPGRADE_SERVICE_URL: whatever');
+      await expect(utils.makeUpgradeRequest(payload))
+        .to.eventually.be.rejectedWith('Invalid UPGRADE_SERVICE_URL: whatever');
     });
 
     it('should throw request errors', async () => {
       const payload = { containers: [], 'docker-compose': {} };
-      sinon.stub(rpn, 'post').rejects(new Error('boom'));
+      sinon.stub(request, 'post').rejects(new Error('boom'));
 
       await expect(utils.makeUpgradeRequest(payload)).to.be.rejectedWith('boom');
     });
 
     it('should throw error when response is invalid', async () => {
       const payload = { docker_compose: { 'doc.yml': 'payload' }, containers: {} };
-      sinon.stub(rpn, 'post').resolves();
+      sinon.stub(request, 'post').resolves();
 
-      await expect(utils.makeUpgradeRequest(payload)).to.be.rejectedWith('No containers were updated');
+      await expect(utils.makeUpgradeRequest(payload)).to.eventually.be.rejectedWith('No containers were updated');
```

Comment:
Why is this change necessary? I expect rpn and @medic/request to behave the same here.

Reply:
This resolved some flakiness on the testing. It's a fix for the testing not the library.

Reply:
That's what I'm worried about - unit test should not be flake because they control their environment. Let's see if we can fix this reliably...

Reply:
I'll revisit. It was intermittent enough of a failure that I might just revert the change.